avtozip/store/static/store/css/main.css

```diff
@@ -0,0 +1,30 @@
+div.div-table {
```

Comment:
I doubt if we need this CSS because we should use BootStrap which you should add to the project